osu.Game/Beatmaps/Formats/LegacyStoryboardDecoder.cs

```diff
@@ -167,150 +171,118 @@ private void handleEvents(string line)
                 if (depth < 2)
                     timelineGroup = storyboardSprite?.TimelineGroup;
 
-                string commandType = split[0];
+                var commandType = span.Slice(ranges[0]);
 
-                switch (commandType)
+                if (commandType.SequenceEqual("T"))
```

Comment:
You can unset the hard coded LangVersion=10 in osu.Game, and use the switch on span feature from C# 11.